### PR TITLE
release v0.3

### PR DESCRIPTION
**Pull request by wvlothuizen**
Since the last release (i.e. update of the master branch) is rather old, it would be good to produce a new release soonish.

This PR has the added benefit of tracking whether CI succeeds on the development branch

### CODE REVIEW

File: pycqed/instrument_drivers/meta_instrument/LutMans/mw_lutman.py [hunk 1]
```diff
@@ -104,13 +244,27 @@ def _add_waveform_parameters(self):
         self.spec_func = wf.block_pulse
 
         self._add_channel_params()
+        # FIXME: Remove after the end of RUS experiment
```
Comment (MiguelSMoreira) [inline pycqed/instrument_drivers/meta_instrument/LutMans/mw_lutman.py:247]: Parameters added from line 247-260 should be removed. These were introduced by mistake and constitute a bug

**Comment by MiguelSMoreira** [comment]
Proposed changelog:
- added support for real-time modulation of waveforms for microwave control
- added support for mixer skewness calibration with real-time modulation
- added support for virtual-Z gates
- added multiple methods for parity check calibration and assessment
- added multiple methods for parallel qubit calibration

**Review by MiguelSMoreira — APPROVED** [review]
With the large number of improvements tracked in develop, and otherwise the large amount of changes encompassed by this pull request, I believe it is a good idea to merge this request, bringing master in sync with develop. The fact that CI is now passing and that the changelog has been updated accordingly, means you should feel free to merge it at your best convinience @wvlothuizen. Thank you for your work on this!

**Comment by wvlothuizen** [comment]
@MiguelSMoreira 
I suggest to add the following few PRs to this release:
- [ ] https://github.com/DiCarloLab-Delft/PycQED_py3/pull/649
- [ ] https://github.com/DiCarloLab-Delft/PycQED_py3/pull/654
- [ ] https://github.com/DiCarloLab-Delft/PycQED_py3/pull/646
- [ ] https://github.com/DiCarloLab-Delft/PycQED_py3/pull/668

And then merge and set a tag.

**Pull request merged**